src/components/timeline/Markers.js

```diff
@@ -50,21 +50,16 @@ type MarkerState = 'PRESSED' | 'HOVERED' | 'NONE';
  */
 
 export type OwnProps = {|
-  +className: string,
   +rangeStart: Milliseconds,
   +rangeEnd: Milliseconds,
   +threadIndex: ThreadIndex,
   +onSelect: any,
 |};
 
 export type StateProps = {|
+  +additionalClassName?: ?string,
   +markers: Marker[],
   +isSelected: boolean,
-  +styles: any,
```

Comment:
The styles were being injected dynamically, but they are statically used. I bet this was to make it more flexible at the beginning, but I don't see any reason to do it this way now.